gocd/bci_repo_publish.py

```diff
@@ -157,8 +182,16 @@ def run(self, version, token=None):
                 return
             openqa_passed_packages.append(pkg)
 
+        if rebuild_token:
+            for pkg in self.fetch_package_names_in_project(_DEVEL_PRJ_APIURL, devel_prj):
+                url = makeurl(_DEVEL_PRJ_APIURL, ['trigger', 'rebuild'], {'project': devel_prj, 'package': pkg})
```

Comment:
this could now be changed based on https://github.com/openSUSE/open-build-service/pull/15261 (which is deployed)